pkg/products/amqstreams/reconciler.go

```diff
@@ -161,21 +162,21 @@ func (r *Reconciler) handleCreatingComponents(ctx context.Context, client k8scli
 		kafka.Name = clusterName
 		kafka.Namespace = r.Config.GetNamespace()
 
-		kafka.Spec.Kafka.Version = "2.1.1"
-		kafka.Spec.Kafka.Replicas = 3
+		kafka.Spec.Kafka.Version = "2.4.0"
```

Comment:
@CathalOConnorRH What is this Kafka version for? is `2.4.0` correct here?

Reply:
If we leave it at 2.1.1, we get errors that the version must be 2.3.0 or 2.4.0. I decided to use 2.4.0 as it would be the newest version to use. 

- 2.3.0=registry.redhat.io/amq7/amq-streams-kafka-23-rhel7:1.4.0
- 2.4.0=registry.redhat.io/amq7/amq-streams-kafka-24-rhel7:1.4.0

Reply:
Sounds good, thanks @CathalOConnorRH